tests/test-files/from-json/Elm-0.19/Expressions.json

```diff
@@ -21,7 +21,7 @@
             "name": "floatLiteral",
             "expression": {
                 "tag": "FloatLiteral",
-                "value": 2
+                "value": 2.0
```

Comment:
This change fixes the tests run by shake.

At a glance, this looks correct, but I'm not sure if something during the upgrade affected this.

Reply:
Hmm, my guess is that maybe this is something that changed in aeson 2.2 for default float deserializers?  I'll have to review how the test suite works here... if this just changes the output, then I think this change is fine.  If this json file is used as input, then I think ideally "2" should still be valid and be allowed to parse.  Maybe the json serializer/deserializer now needs to be manually specified?  Also I wonder if now aeson needs to be locked to `>= 2.2` if using 2.1 vs 2.2 does actually change the behavior.

Reply:
Thanks for the context. I'll try to narrow down what caused this.

Reply:
Strange, if I `cabal freeze` aeson to `2.1.2.1` the tests still expect `2.0`. Not really sure what's going on.